browser/src/control/Control.Sidebar.js

```diff
@@ -146,7 +146,8 @@ L.Control.Sidebar = L.Control.extend({
 	},
 
 	changeDeck: function(unoCommand) {
-		app.socket.sendMessage('uno ' + unoCommand);
+		if (unoCommand !== null)
+			app.socket.sendMessage('uno ' + unoCommand);
 		this.setupTargetDeck(unoCommand);
```

Comment:
If `unoCommand == null`, should we still call `this.setupTargetDeck` ?

Reply:
yes, we want to reset it